src/frontend/components/SøknadsSteg/OmBarnetUtfyllende/OmBarnetUtfyllende.tsx

```diff
@@ -0,0 +1,126 @@
+                                <FormattedMessage
+                                    id={
+                                        OmBarnetUfyllendeSpørsmålSpråkId[
+                                            OmBarnetUfyllendeSpørsmålsId.institusjonsnavn
+                                        ]
+                                    }
+                                />
+                            }
+                            {...skjema.felter.institusjonsnavn.hentNavInputProps(
+                                skjema.visFeilmeldinger
+                            )}
+                        />
+                    </>
+                )}
+                {skjema.felter.institusjonsnavn.erSynlig && (
+                    <Input
+                        label={
+                            <FormattedMessage
+                                id={
+                                    OmBarnetUfyllendeSpørsmålSpråkId[
+                                        OmBarnetUfyllendeSpørsmålsId.institusjonsadresse
+                                    ]
+                                }
+                            />
+                        }
+                        {...skjema.felter.institusjonsadresse.hentNavInputProps(
+                            skjema.visFeilmeldinger
+                        )}
+                    />
+                )}
+                {skjema.felter.institusjonsnavn.erSynlig && (
+                    <Input
+                        bredde={'S'}
+                        label={
+                            <FormattedMessage
+                                id={
+                                    OmBarnetUfyllendeSpørsmålSpråkId[
+                                        OmBarnetUfyllendeSpørsmålsId.institusjonspostnummer
+                                    ]
+                                }
+                            />
+                        }
+                        {...skjema.felter.institusjonspostnummer.hentNavInputProps(
+                            skjema.visFeilmeldinger
+                        )}
+                    />
+                )}
+            </KomponentGruppe>
+        </Steg>
+    );
+};
+
+const OmBarnetUtfyllende: React.FC = () => {
+    const { søknad } = useApp();
+    const { barnInkludertISøknaden } = søknad;
+    const history = useHistory();
+    const location = useLocation();
+    const match = location.pathname.match(/\/barnet\/(.+)\/?/);
+
+    if (!barnInkludertISøknaden.length) {
+        history.push('/velg-barn');
+        return <></>;
+    }
+
+    if (!match || match.length < 2) {
+        history.push(`/barnet/${slugify(barnInkludertISøknaden[0].navn)}`);
+        return <></>;
+    }
+    const navnParameter = match[1];
+
+    const barn = barnInkludertISøknaden.find(barn => slugify(barn.navn) === navnParameter);
+    if (!barn) {
+        history.push('/barnet');
+        return <></>;
+    }
+
+    return <InternKomponent barn={barn} />;
```

Comment:
Synes det hadde vært mer oversiktlig hvis denne ikke var wrappet i en interkomponent kanskje 🙈  pga. av den kun returnerer én linje, og jeg er vant med at det kan være en del i return på en måte. Men det er personlig preferanse da, tenkte bare å lufte det så kan gjøre med det som du vil 😊

Reply:
Eneste grunnen til at jeg har den i en internkomponent er fordi react skriker til meg om inkonsistente props hvis jeg inliner den :)

Reply:
Sorry, ikke props. Her er feilen:
`React Hook "useOmBarnetUtfyllende" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?  react-app/react-hooks/rules-of-hooks`

Reply:
Ah, skjønner! alle hooks må stå på toppen i komponenten hvis man ikke skulle wrappet den.